### PR TITLE
DEVDOCS-5953: [update] add date created and date modified

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-5953]

## What changed?
Added parameters to the GET batch metafields endpoint.

## Release notes draft

* 

## Anything else?

ping {names}

[DEVDOCS-5953]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5953?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by alex-gaiduchok** [comment]
@bc-rafsierk please take a look

File: reference/locations.v3.yml [hunk 1]
```diff
@@ -1731,6 +1735,38 @@ components:
         enum:
           - asc
           - desc
+    date_created_min:
+      name: 'date_created:min'
+      description: |
+        'Query parameter that lets you filter by the minimum date created, for example, `2024-05-14T09:34:00Z`. Returns metafields created after this date.'
```
Comment (bc-rafsierk) [inline reference/locations.v3.yml:1741]: maybe `Query parameter that lets you filter by the date created which is before provided one`

FYI: it works like `>=`

File: reference/locations.v3.yml [hunk 2]
```diff
@@ -1731,6 +1735,38 @@ components:
...
+      required: false
+      in: query
+      schema:
+        type: string
+    date_created_max:
+      name: 'date_created:max'
+      description: |
+        'Query parameter that lets you filter by the maximum date created, for example, `2024-05-14T09:34:00Z`. Returns metafields created before this date.'
```
Comment (bc-rafsierk) [inline reference/locations.v3.yml:1749]: As previously I would state that `Query parameter that lets you filter by the date created which is before provided one`
FYI: it works similarly to `<=`

File: reference/locations.v3.yml (hunk 1 above)
Comment (bc-rafsierk) [inline reference/locations.v3.yml:1741]: Do we want to mention that we can provide just a date as well ?

Comment (bc-traciporter) [reply reference/locations.v3.yml:1741]: We can, are you suggesting that we add a `date_created` parameter to the list? Would I need to do this just for the locations file or for the other APIs in [PR 322](https://github.com/bigcommerce/docs/pull/322/files).

**Review by bc-traciporter — COMMENTED** [review]
@bc-rafsierk I left questions

File: reference/locations.v3.yml (hunk 1 above)
Comment (bc-rafsierk) [reply reference/locations.v3.yml:1741]: no, just if I would be a api user I would have no idea what `minimum date created` is. What I am suggesting is to say that this `min` means that the date provided in the filter will return metafields with the dates which are `after` the one used in filter. 
But if we usually just write minimum I am fine with that as well

Comment (bc-traciporter) [reply reference/locations.v3.yml:1741]: Hi @bc-rafsierk , How does this sound? Is this clear?
Query parameter that lets you filter by the minimum date created. The date provided in the filter will return metafields after this date. Example format = 2024-05-14T09:34:00Z.

**Review by bc-rafsierk — APPROVED** [review]

**Review by bc-andreadao — APPROVED** [review]

**Pull request merged**